Cargo.toml

```diff
@@ -27,7 +27,7 @@ num_cpus = { version = "1.13", optional = true }
 which = { version = "5.0", optional = true }
 
 [target.'cfg(not(target_os = "windows"))'.dependencies]
-users = { version = "0.11", optional = true }
+uzers = { version = "0.11.3", optional = true }
```

Comment:
Specifying the "patch" version makes cargo to resolve the `uzers` dependency with `>= v0.11.3` constraint.